### ISSUE TITLE
Helm-init-candidates-in-buffer: Wrong type argument error when compiling a TeX file

### ISSUE THREAD

**Issue by patrl** [syl20bnr/spacemacs#8793]
#### Description :octocat:
Helm-init-candidates-in-buffer: Wrong type argument error when compiling a TeX file

#### Reproduction guide :beetle:
- Start Emacs
- Open a `.tex` file
- Run `TeX-command-master` 

*Observed behaviour:* :eyes: :broken_heart:
Spacemacs throws out the following error:

```
helm-init-candidates-in-buffer: Wrong type argument: characterp, "LatexMk"
```

*Expected behaviour:* :heart: :smile:

Helm should provide a list of tex commands and auto-select the appropriate one.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: master (rev. 8e1af145)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(bibtex python javascript helm auto-completion better-defaults colors emacs-lisp git github haskell html latex markdown deft pandoc org
        (shell :variables shell-default-height 30 shell-default-position 'bottom)
        spell-checking syntax-checking mu4e writeroom ranger)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES

**Comment by maxbeegee** [comment]
I had the same problem and resolved it by hard resetting to v0.200.9. Then after starting I updated all my packages (helm included) and it worked again. Which version of ac-helm are using?

**Comment by patrl** [comment]
This is solved by the latest helm update. Closing!
